2021-01-18 18:40:16,254 | ERROR   | [Errno 13] Permission denied: '/home/danilo/Documents/ProjetosPython/status.sh'
Traceback (most recent call last):
  File "/home/danilo/Documents/ProjetosPython/botTelegram.py", line 48, in send_server
    call(["/home/danilo/Documents/ProjetosPython/status.sh"])
  File "/usr/lib64/python3.9/subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib64/python3.9/subprocess.py", line 947, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib64/python3.9/subprocess.py", line 1819, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
PermissionError: [Errno 13] Permission denied: '/home/danilo/Documents/ProjetosPython/status.sh'
